docs/generic-storage.md

```diff
@@ -63,52 +63,49 @@ To get/put specific object collection instance should be used.
 Collection instance provide set of methods to handle data objects.
 List of collection methods:
 ```
-def put(self, object_id: str, object_body: JSON_TYPES) -> None:
-    """
-    Put object into collection.
+    def put(self, object_id: str, object_body: JSON_TYPES) -> None:
```

Comment:
I added indentation so we can test consistency by string inclusion, without post-processing of the string.